.npmrc

```diff
@@ -1,2 +1,4 @@
 registry=https://registry.npmjs.org/
 save-exact=true
+update-notifier=false
+auto-install-peers=true
```

Comment:
It can hide missing peer dependency issues and make installs less predictable. Why is that better here than keeping peer deps explicit?

Reply:
This setting only affects development, and packages won't appear in root of `node_modules` and won't be able to be imported. Therefore, this won't cause any problems